### PR TITLE
BrenKt-SK900-TRANS-WEB-Emails-all_offers-manage_overquota

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web/emails/email_manage_quota/guide.it-it.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+Se il tuo account email è saturo, significa che non sei più in grado di ricevere email.<br>
+Quando una persona ti invia un'email, riceve, in risposta automatica, un'email di errore del tipo *"552, "5.2.2", l'account di posta a cui hai inviato un messaggio ha esaurito la sua quota. "*.<br>
+Quando il tuo account email è saturo, puoi sempre inviare email dalla tua parte. Queste email, invece, non potranno essere salvate nella tua "casella di invio".
+
+#### Ottimizza lo spazio associato al tuo account email
+
+Prima di effettuare qualsiasi altra operazione sul tuo account email, è necessario prendere visione del contenuto del tuo account per eliminare tutti gli elementi superflui. Ti consigliamo di verificarne alcuni in particolare:
+
+- `1`{.action} **La Corbeille (Trah)**: contiene gli elementi che hai eliminato. Per evitare di accumulare email in questa cartella, ti consigliamo di svuotare regolarmente il cestino.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_manage_quota/guide.it-it.md]: Looks like FR content here

File: pages/web/emails/email_manage_quota/guide.pt-pt.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+>> - [Thunderbird](https://docs.ovh.com/pt/microsoft-collaborative-solutions/configuracao-exchange-thunderbird-mac/) (gratuito)<br>
+>>
+
+Depois de instalar o seu software de correio, siga as instruções abaixo para preparar a pasta do arquivo no seu software de correio eletrónico.
+
+> [!tabs]
+> **Outlook**
+>>
+>> No Outlook, certifique-se de que, na sua coluna da esquerda, a pasta "arquivo" ou "no meu computador" está presente para poder introduzir os elementos que deseja manter localmente no seu computador. Consulte a documentação da Microsoft para preparar a sua pasta de arquivo:<br><br>
+>> - [Arquivo no Outlook para Windows](https://support.microsoft.com/pt-pt/office/arquivar-no-outlook-para-windows-25f75777-3cdc-4c77-9783-5929c7b47028) {.external}<br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_manage_quota/guide.pt-pt.md]:
```suggestion
>> - [Arquivo no Outlook para Windows](https://support.microsoft.com/pt-pt/office/arquivar-no-outlook-para-windows-25f75777-3cdc-4c77-9783-5929c7b47028){.external}<br>
```

File: pages/web/emails/email_manage_quota/guide.pt-pt.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+>> - [Sobre as pastas do meu computador no Outlook para Mac](https://support.microsoft.com/pt-pt/office/acerca-das-pastas-no-meu-computador-no-outlook-para-mac-c91b8729-924d-4c25-a5f6-38883d0f763d) {.external}<br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_manage_quota/guide.pt-pt.md]:
```suggestion
>> - [Sobre as pastas do meu computador no Outlook para Mac](https://support.microsoft.com/pt-pt/office/acerca-das-pastas-no-meu-computador-no-outlook-para-mac-c91b8729-924d-4c25-a5f6-38883d0f763d){.external}<br>
```

File: pages/web/emails/email_manage_quota/guide.pt-pt.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+>>
+> **Mail macOS**
+>>
+>> A partir do e-mail no macOS, crie uma pasta que aparecerá na secção "No meu Mac" na coluna da esquerda. Para isso, siga a documentação da Apple:<br><br>
+>> - [Criar ou eliminar caixas de correio no Mail](https://support.apple.com/pt-pt/guide/mail/mlhlp1021/15.0/mac/12.0) {.external}<br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_manage_quota/guide.pt-pt.md]:
```suggestion
>> - [Criar ou eliminar caixas de correio no Mail](https://support.apple.com/pt-pt/guide/mail/mlhlp1021/15.0/mac/12.0){.external}<br>
```

File: pages/web/emails/email_manage_quota/guide.pt-pt.md [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+>>
+> **Thunderbird**
+>>
+>> Através do thunderbird a partir do Windows, macOS ou Linux, pode mover os seus e-mails para uma pasta da parte esquerda. Ajude-se com a documentação de Mozilla:<br><br>
+>> - [Arquivo de mensagens](https://support.mozilla.org/pt-PT/kb/mensagens-arquivadas) {.external}<br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_manage_quota/guide.pt-pt.md]:
```suggestion
>> - [Arquivo de mensagens](https://support.mozilla.org/pt-PT/kb/mensagens-arquivadas){.external}<br>
```

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**